common_package_py/commands.py

```diff
@@ -0,0 +1,344 @@
+import json
+import rclpy
+from enum import Enum
```

Comment:
Int enum & Unique & Optional & Union
```suggestion
from enum import IntEnum, Unique
from typing import Optional, Union
```

Reply:
ERROR  - ImportError: cannot import name 'Unique' from 'enum' (/usr/lib/pytho...
!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!

Reply:
Schreibt man nämlich klein: `unique`